packages/react/src/components/Checkbox/Checkbox-test.js

```diff
@@ -190,3 +193,35 @@ describe('CheckboxSkeleton', () => {
     });
   });
 });
+
+describe('Checkbox accessibility', () => {
+  afterEach(cleanup);
+
+  it('should have no Axe violationk', async () => {
+    render(<Checkbox labelText="Checkbox label" id="test_id" />);
+    await expect(screen.getByText('Checkbox label')).toHaveNoAxeViolations();
+  });
+
+  it('should have no Accessibility Checker violations', async () => {
+    render(<Checkbox labelText="Checkbox label" id="test_id" />);
+    await expect(screen.getByText('Checkbox label')).toHaveNoACViolations(
+      'Checkbox'
+    );
+  });
+
+  it('can receive keyboard focus', () => {
+    render(<Checkbox labelText="Checkbox label" id="test_id" />);
+    userEvent.tab();
+    expect(screen.getByLabelText('Checkbox label')).toHaveFocus();
+  });
+
+  it('should have an accessible label', () => {
+    render(<Checkbox labelText="Checkbox label" id="test_id" />);
+    expect(screen.getByText('Checkbox label'));
+  });
+
+  it('should have an appropriate role', () => {
+    render(<Checkbox labelText="Checkbox label" id="test_id" />);
+    expect(screen.getByRole('checkbox'));
```

Comment:
This might be missing an assertion part to the expect? Let me know if I'm mis-reading 👀